### PR TITLE
first commit for new function

### PR DESCRIPTION
**Pull request by jd-lara**
Thanks for opening a PR to PowerNetworkMatrices.jl, please take note of the following when making a PR:

Check the [contributor guidelines](https://nrel-siip.github.io/PowerNetworkMatrices.jl/stable/code_base_developer_guide/developer/)

### CODE REVIEW

File: test/case_ACTIVSg10k.m
Comment (jd-lara) [inline test/case_ACTIVSg10k.m:1]: Please remove this files, test files do not belong in the PR

File: test/case_ACTIVSg70k.m
Comment (jd-lara) [inline test/case_ACTIVSg70k.m:1]: Please remove this files, test files do not belong in the PR

**Review by jd-lara — COMMENTED** [review]
@alefcastelli please do not add test files to the PR

File: src/PowerNetworkMatrices.jl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module PowerNetworkMatrices
 
 export PTDF
+export getdata
```
Comment (jd-lara) [inline src/PowerNetworkMatrices.jl:4]:
```suggestion
export get_data
```

File: test/case_ACTIVSg10k.m
Comment (jd-lara) [inline test/case_ACTIVSg10k.m:1]: remove testing files

File: test/test_ABA_matrix.jl [hunk 2]
```diff
@@ -17,7 +17,7 @@
         # evaluate if the ABA matrix is correct
         A = IncidenceMatrix(sys)
         BA = BA_Matrix(sys)
-        ABA_2 = transpose(A.data) * BA.data
+        ABA_2 = transpose(A.data) * BA.data'
```
Comment (jd-lara) [inline test/test_ABA_matrix.jl:20]: why transpose twice?

**Review by jd-lara — COMMENTED** [review]
@alefcastelli please remove the testing files

File: test/test_ABA_matrix.jl (hunk 2 above)
Comment (alefcastelli) [reply test/test_ABA_matrix.jl:20]: because ABA = A'*BA. Since the new new BA is evaluated as a transpose matrix, then ABA = A' * BA'
